api/tests/test.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+const os = require('os')
+const path = require('path')
+const fs = require('fs-extra')
+const nock = require('nock')
+const test = require('ava') //eslint-disable-line import/no-unresolved
```

Comment:
the eslint-disable-line is not necessary. ava is added as dependency and should be found by the linter